src/main/java/net/minecraftforge/registries/DataPackRegistriesHooks.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.registries;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.google.common.collect.ImmutableMap;
+
+import net.minecraft.core.Registry;
+import net.minecraft.core.RegistryAccess;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.resources.ResourceLocation;
+
+public class DataPackRegistriesHooks
```

Comment:
Is there anything in here which may be used by modders?

Reply:
Only the view getter is safe to use, and I'm not if there's any use case to use it in userdev -- I can mark the whole class as forge-internal, in any case